guildwars2/guild/general.py

```diff
@@ -17,23 +17,31 @@ class GeneralGuild:
 
     @guild.command(name="info")
```

Comment:
add `usage="<guild name>"` in the decorator to this and other commands too, else help message is ugly

Reply:
Added to guild commands